src/App.tsx

```diff
@@ -1,42 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
+import { Tabs } from './components/Tabs';
+
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0]; 
```
Let the first tab be selected if no other tab is found